src/components/earn/PoolCardTri/ClaimRewardModalTri.tsx

```diff
@@ -84,7 +98,7 @@ export default function ClaimRewardModal({ isOpen, onDismiss, stakingInfo }: Sta
   if (!account) {
     error = t('earn.connectWallet')
   }
-  if (!stakingInfo?.stakedAmount) {
+  if (!stakedAmount) {
```

Comment:
Should this be an `else if`?  Account should have higher priority than staked amount?

Reply:
Changed! Tbh in this comp I only removed staking info, so didn't even notice this  but good thought :)